introduction.md

```diff
@@ -1 +1,3 @@
 https://www.codeschool.com/paths/ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?